tests/nnapi/specs/V1_0/resize_bilinear_nnfw.mod.py

```diff
@@ -2,9 +2,10 @@
 model = Model()
 i1 = Input("op1", "TENSOR_FLOAT32", "{1, 2, 2, 1}")
 i2 = Output("op2", "TENSOR_FLOAT32", "{1, 3, 3, 1}")
-w = Int32Scalar("width", 3)
-h = Int32Scalar("height", 3)
-model = model.Operation("RESIZE_BILINEAR", i1, w, h).To(i2)
+s = Parameter("size", "TENSOR_INT32", "{2}",[3, 3])
+a = Int32Scalar("align_corners", 0)
+h = Int32Scalar("half_pixel_centers", 0)
+model = model.Operation("RESIZE_BILINEAR", i1, s, a, h).To(i2)
```

Comment:
Don't change NNAPI test. And this change is invalid with NNAPI spec.

Reply:
Thanks for the explanation. I have since then made the changes.